pop-api/README.md

```diff
@@ -1,142 +1,64 @@
-# Pop API
+## Pop API
 
+The `pop-api` crate provides a high-level interface that allows smart contracts to seamlessly interact with Pop, a
+blockchain built to power innovative and impactful solutions on Polkadot. Designed for stability, simplicity and
+efficiency, the API abstracts away the complexities of the runtime, enabling developers to focus on building powerful
+applications rather than managing intricate blockchain details.
 
-- [What is Pop API](#what-is-pop-api)
-- [Versions](#versions)
-- [Use Pop API](#use-pop-api)
-- [Examples](#examples)
-- [Testing](#testing)
-- [Design of Pop API](#design-of-pop-api-)
-- - [The Pop API ink! Library](#the-pop-api-ink-library)
-- - [The Glue](#the-glue)
-- - [The Entry into the Pop API](#the-entry-into-the-pop-api)
-- - [Dispatching to the runtime](#dispatching-to-the-runtime-srclibrs)
-- - [The Pop API Chain Extension](#the-pop-api-chain-extension)
+### Design Goals
 
----
+- **Simple**: enhance the developer experience by abstracting away the complexities of runtime functionality, making it
+  easy for developers to build advanced applications.
+- **Versioned**: offer a stable, versioned interface that ensures smart contracts stay compatible as the runtime
+  evolves, enabling seamless integration of new features without disrupting existing contracts.
+- **Efficient**: optimise for minimal contract size, having the lowest possible deployment and execution costs.
 
-## What is Pop API
+### Key Features
 
-The Pop API ink! library provides APIs for tailored use cases. The apis are versioned ensuring compatibility across different runtime updates and preserves the functionality of existing contracts.
+- **Versioned Interface**: Provides backward compatibility, ensuring that existing contract functionality remains stable
+  as new features are added to the runtime.
+- **Error Handling**: Supports rich, versioned error types, enabling contracts to receive and interpret any runtime
+  error, making troubleshooting and development easier.
+- **Use Cases**:
+    - [Fungibles](./src/v0/fungibles/README.md): Interacting and managing fungible tokens.
+    - Coming soon:
```

Comment:
I am not sure we should include this here. There might be better ways of communicating a feature roadmap than in this README.

Until we have something better, I believe it would be better indicating if the specific feature is integrated the API now or it is in development instead of Dec / TBD / Exploring.

Reply:
So what are you saying? `In development`?

Reply:
Yeah, that sounds good to me